lib/Twig/Node/Expression/Call.php

```diff
@@ -259,14 +259,24 @@ private function reflectCallable($callable)
         }
 
         if (is_array($callable)) {
+            if (!method_exists($callable[0], $callable[1])) {
+                // __call()
+                return;
```

Comment:
returning null will break the compilation. ``compileCallable`` and ``getCallableParameters`` expect an array with 2 elements